crates/rustc_codegen_spirv/src/linker/structurizer.rs

```diff
@@ -518,10 +557,7 @@ pub fn insert_selection_merge_on_conditional_branch(
                 return;
             } else {
                 // (fully unreachable) insert a rando block and mark as merge.
-                if let Some(sess) = sess {
-                    sess.err("UNIMPLEMENTED, A fully unreachable case was detected.");
-                }
-                return;
+                blocks[b_first_idx].label_id().unwrap()
```

Comment:
This comment doesn't seem to match up with the implementation, I'm confused.

Reply:
Oh oops. That was remaining from a test.